### ISSUE TITLE
s:vimwiki_autowriteall initialization

### ISSUE THREAD

**Issue by sdothum** [vimwiki/vimwiki#174]
Updated to vimwiki release 2.2 and encountered this error when changing buffers (open vim with several vimwiki indexes) for the first time:

```
Error detected while processing function <SNR>80_setup_buffer_leave:
line   12:
E121: Undefined variable: s:vimwiki_autowriteall
E15: Invalid expression: s:vimwiki_autowriteall
```

After ignoring the error, subsequent buffer changes do not post any further error messages.

Copying (from the vimwiki.vim source) and pasting to the end of the vimviwki.source:

```
if !exists("s:vimwiki_autowriteall")
  let s:vimwiki_autowriteall = &autowriteall
endif
```

cleans up what appears to be an initialization problem (on my system, at least). Not sure if this is the correct code or best location but it seems to work.

**Comment by EinfachToll** [comment]
Although I have never seen it and can't reproduce it, I think this related to the way Vimwiki handles its internal state, which is a constant source of spurious bugs. This is currently a huge mess, but I'm after it.

**Comment by ranebrown** [comment]
This should be fixed with #256. If you are still having issues please try the `dev` branch and then reopen.
